models/edr/alerts/alerts_dbt_source_freshness.sql

```diff
@@ -0,0 +1,15 @@
+{{
+  config(
+    materialized = 'view',
+    bind=False
+  )
+}}
+
+select source_freshness_execution_id as alert_id,
+       unique_id,
+       max_loaded_at
+       snapshotted_at as detected_at,
```

Comment:
1. Consider adding the join with sources here
2. Add `generated_at`
3. The alias for snapshotted_at should not be detected_at as it's gonna be confusing with `generated_at`. So here I'd use the `snapshotted_at` and in the alerts itself let's call it `source freshness test time`